react/src/lib/components/DeckGLMap/DeckGLMap.tsx

```diff
@@ -28,6 +28,10 @@ interface DeckGLMapProps {
         widthPerUnit?: number | null;
         position?: number[] | null;
     };
+    northArrow?: {
+        visible?: boolean | null;
+        position?: number[] | null;
```

Comment:
Would be nice also to optionally be able to override the GLTF file, in order to be able to a custom styled north arrow.

Reply:
Could we do that in a later commit?